scripts/build.ps1

```diff
@@ -95,12 +98,25 @@ param (
     [switch]$DynamicCRT = $false,
 
     [Parameter(Mandatory = $false)]
-    [switch]$PGO = $false
+    [switch]$PGO = $false,
+
+    [Parameter(Mandatory = $false)]
+    [string]$Generator = ""
 )
 
 Set-StrictMode -Version 'Latest'
 $PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
 
+if ($IsWindows -and ($Generator -ne "")) {
+    Write-Error "Generator setting not supported on windows"
```

Comment:
Why not default to `Visual Studio 16 2019`?

Reply:
I can default to that, but theres the arch arguments to which have to be set afterwards. I can do that by adding a second variable I guess.

Reply:
Just implicitly do the arch argument. Just let this change the generator.

Reply:
Fixed. Should be good to go now.